src/components/command/commandComponent/CommandComponentLayout.tsx

```
@@ -0,0 +1,99 @@
+    isShowReply,
+    handleShowReplyClick,
+    handleDropDownClick,
+    setActivateEditMode,
+    handleActivateClick,
+    onEdit,
+  } = useCommand();
+
+  return (
+    <>
+      {getCommandList?.map((item, index) => (
+        <S.Container>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)